pages/en/lb3/HasManyThrough-relations.md

```diff
@@ -77,6 +101,25 @@ For example:
   ...
 ```
 
+You will execute two more lb relations for the belongsTo relation
```

Comment:
_Suggest slight rewording_:

Then, in the Physician/Patient/Appointment example, you would run the `lb relation` command twice to create the `belongsT`o relations: